src/Mailjet/Message/AdherentLeaveMembershipMessage.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace AppBundle\Mailjet\Message;
+
+use AppBundle\Entity\Adherent;
+use Ramsey\Uuid\Uuid;
+
+final class AdherentLeaveMembershipMessage extends MailjetMessage
```

Comment:
The word `leave` looks weird to me. Better words might be `close`, `finish`, `terminate`, `end`, `shutdown`.